server/planning/search/planning_search.py

```diff
@@ -63,7 +108,7 @@ def get(self, req, lookup):
 
             params["_source"] = fields
 
-        docs = self.elastic.search(query, types, params)
+        docs = await self.elastic.search_async(query, types, params)
```

Comment:
Elastic async support is not yet added to Eve datalayer. What you could use instead is the Events resource to get a client handle for Elastic (both Events & Planning run on the same es instance).

```python
from planning.types import EventResourceModel

async def get_async(self, req, lookup):
    ...
   docs = await EventResourceModel.get_service().elastic.search(query, types, params)
```

See how that goes

Reply:
I checked the definition of the search and it takes a query, and indexes : 
```    
async def search(self, query: Dict[str, Any], indexes: Optional[List[str]] = None) -> Any:
```

I can get the indexes using the `_get_index` in the class but I'm not so sure what to do with the `types` and `fields` as they were required in the previous search implementation. 

Thoughts ?